parm/MOM_input_template_025

```diff
@@ -783,6 +783,29 @@ LT_ENHANCE = 3                  !   [nondim] default = 0
                                 !          1 - Van Roekel et al. 2014/Li et al., 2016
                                 !          2 - Multiplied w/ adjusted La.
                                 !          3 - Added w/ adjusted La.
+USE_WAVES = MOM6_USE_WAVES      !   [Boolean] default = False
```

Comment:
Will this input template only be used for tests with ww3 or it can also be used by test without ww3?

Reply:
It can be used for either waves or no waves, depending on whether MOM6_USE_WAVES is true or false in the test.